zoom/plugin.spec.yaml

```diff
@@ -355,3 +355,50 @@ triggers:
         description: User Activity
         type: user_activity
         required: false
+events:
```

Comment:
Is there something not specific to `events` and more generic we could use that properly defines "code that runs on schedule that (usually) returns data"?  Perhaps `scheduled_task` or `scheduled_action`? @jonschipp @mrinehart-r7 @ostewart-r7

Reply:
I'm partial to `events` tbh. Logs / alerts are fit under events IMO

Reply:
Maybe `collection` are some synonym that means to retrieve

Reply:
They way I look at it is this is just a generic "job" that does something.  Right now we think of it as collect events or data for triggers, etc but what if it is more generically as "run this job every X often"?

Reply:
Or what about `tasks`?